### PR TITLE
Dml batch update request priority

### PR DESCRIPTION
**Pull request by ajupazhamayil**
This PR creates a sample for DML batch update request with low priority.

### CODE REVIEW

File: spanner/src/add_drop_database_role.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+ * Adds and drops roles to the Singers table in the example database.
+ * Example:
+ * ```
+ * add_drop_database_role($instanceId, $databaseId);
+ * ```
+ *
+ * @param string $instanceId The Spanner instance ID.
+ * @param string $databaseId The Spanner database ID.
+ */
+function add_drop_database_role(string $instanceId, string $databaseId): void
```
Comment (vishwarajanand) [inline spanner/src/add_drop_database_role.php:39]: This sample seems to be from: https://github.com/GoogleCloudPlatform/php-docs-samples/pull/1689. 
Consider creating a new git branch (base to `master`) with just the intended changes and force push to `dml_batch_update_request_priority` branch.

File: spanner/test/spannerTest.php [hunk 2]
```diff
@@ -802,6 +802,28 @@ public function testCreateClientWithQueryOptions()
...
+        $this->assertStringContainsString('Revoked privileges and dropped roles new_child and new_parent ' . PHP_EOL, $output);
+    }
+
+    /**
+     * @depends testAddColumn
+     */
+    public function testSpannerDmlBatchUpdate()
+    {
+        $output = $this->runFunctionSnippet('dml_batch_update_request_priority');
+        self::$lastUpdateDataTimestamp = time();
```
Comment (vishwarajanand) [inline spanner/test/spannerTest.php:823]: @ajupazhamayil Will `testReadStaleData` fail when this is not set?

**Comment by vishwarajanand** [comment]
LGTM.
Explore on how to sanitize the PR (only relevant commits/files should be there) and I left a question for you.

File: spanner/test/spannerTest.php (hunk 2 above)
Comment (ajupazhamayil) [reply spanner/test/spannerTest.php:823]: `testUpdateData` has the required line to set the `lastUpdateDataTimestamp variable as follows:

'
    public function testUpdateData()
    {
        $output = $this->runFunctionSnippet('update_data');
        self::$lastUpdateDataTimestamp = time();
        $this->assertEquals('Updated data.' . PHP_EOL, $output);
    }
'

File: spanner/src/add_drop_database_role.php (hunk 1 above)
Comment (ajupazhamayil) [reply spanner/src/add_drop_database_role.php]: Done!

File: spanner/test/spannerTest.php (hunk 2 above)
Comment (ajupazhamayil) [reply spanner/test/spannerTest.php:811]: Understood remove 823 line and check the test passes

Comment (ajupazhamayil) [reply spanner/test/spannerTest.php]: Done!

**Pull request merged**